src/mgos_ccs811.c

```diff
@@ -21,14 +21,31 @@
 // Datasheet:
 // https://ams.com/eng/content/download/951091/2269479/file/CCS811_DS000459_4-00.pdf
 
+bool i2c_read_reg_n(struct mgos_i2c *conn, uint16_t addr, uint8_t reg,
+                         size_t n, uint8_t *buf) {
+  const bool wres = mgos_i2c_write(conn, addr, &reg, 1, false /* stop */);
+  mgos_usleep(50);
```

Comment:
can you add a // NOTE() here on why you inserted the sleep?

Reply:
thanks, will work on this change